script_perf_eval.py

```
@@ -0,0 +1,927 @@
+    baseline_path: Path,
+    session_path: Path,
+    *,
+    obj_atol: float,
+) -> None:
+    baseline = json.loads(baseline_path.read_text(encoding="utf-8"))
+    session = json.loads(session_path.read_text(encoding="utf-8"))
+    print("\n" + "=" * 50)
+    print("Cross-process objective check (baseline vs session)")
+    print("=" * 50)
+    all_ok = True
+    for b_rec, s_rec in zip(baseline["objective_records"], session["objective_records"]):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)